marlowe/src/Language/Marlowe/Scripts.hs

```diff
@@ -38,6 +38,12 @@
 
 {-# OPTIONS_GHC -Wno-name-shadowing #-}
 
```

Comment:
Cosmetics: Should we mention this commented pragmas section in the `benchmarking/ReadMe.md` above?

Reply:
I'm not sure. It's a pretty esoteric use case.